### PR TITLE
Story/tyta 109/add view for creating team

### PR DESCRIPTION
**Pull request by kilianpichard**

### CODE REVIEW

File: src/pages/Teams/ListTeams/ListTeams.jsx [hunk 1]
```diff
@@ -43,11 +43,7 @@ const ListTeams = () => {
           variant="outlined"
           href="/teams/add"
           startIcon={
-            <GroupAddIcon
-              sx={{
-                padding: '0 10px 0 0',
-              }}
-            />
+            <GroupAddIcon />
```
Comment (remirubis) [inline src/pages/Teams/ListTeams/ListTeams.jsx:46]: Good job I didn't know

File: src/pages/Teams/AddTeam/AddTeam.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const dispatch = useDispatch();
+
+  const addTeam = () => {
+    if (name.trim().length > 0) {
+      dispatch(createTeam(name));
+    } else {
+      dispatch({
+        type: SET_NOTIFICATION,
+        payload: {
+          message: 'Nom de team incorrect !',
```
Comment (remirubis) [inline src/pages/Teams/AddTeam/AddTeam.jsx]: Can be remained in English as we seem to make all the app in English, I still do not know why mdrr

File: src/pages/Teams/AddTeam/AddTeam.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        sx={{
+          margin: '20px 0',
+        }}
+      >
+        <TextField
+          id="team-name"
+          label="Name"
+          value={name}
+          onChange={handleName}
+        />
```
Comment (remirubis) [inline src/pages/Teams/AddTeam/AddTeam.jsx:51]: We can add a field for the description of the team, right?

**Review by remirubis — COMMENTED** [review]
Just a small change and a suggestion :)

File: src/pages/Teams/AddTeam/AddTeam.jsx (hunk 3 above)
Comment (kilianpichard) [reply src/pages/Teams/AddTeam/AddTeam.jsx:51]: there is none in the model currently ^^

**Review by remirubis — DISMISSED** [review]
LGTM! 🚀 (We can add description in the future if we wants but not urgent thing)

**Pull request merged**